dev/format.sh

```diff
@@ -1,92 +0,0 @@
-#!/bin/bash
```

Comment:
Can we keep this script but use it solely for formatting? 

It's a much easier local workflow to run `./dev/format.sh` than to setup and run `act` -- especially so for new contributors.